src/components/ProjectionResult.js

```diff
@@ -1,52 +1,118 @@
 import { translate } from '@geops/tree-lib';
-import React from 'react';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import { Divider, Form, Header, Tab } from 'semantic-ui-react';
 
+import { setRecommendationMode } from '../store/actions';
 import Recommendation from './Recommendation';
 
 function ProjectionResult() {
-  const { projectionLocation } = useSelector(state => ({
+  const dispatch = useDispatch();
+  const { projectionLocation, location } = useSelector(state => ({
     projectionLocation: state.projectionLocation,
+    location: state.location,
   }));
   const { t, i18n } = useTranslation();
 
+  const [future, setFuture] = useState(false);
+
   const panes = [
     {
-      menuItem: t('tab.scenario1'),
-      render: () => <Tab.Pane>Coming soon...</Tab.Pane>,
+      menuItem: t('recommendationMode.today'),
+      recommendationMode: 'today',
+      render: () => (
+        <Tab.Pane>
+          {' '}
+          <Header>
+            {`${location.forestType} - ${translate(
+              'forestType',
+              location.forestType,
+              i18n.language,
+            )}`}
+          </Header>
+          <Divider hidden />
+          <Recommendation forestTypeToday={location.forestType} />
+        </Tab.Pane>
+      ),
     },
     {
-      menuItem: t('tab.scenario2'),
+      menuItem: t('recommendationMode.moderate'),
+      recommendationMode: 'moderate',
       render: () => (
         <Tab.Pane>
           <Header>
-            {translate(
+            {`${projectionLocation.forestType} - ${translate(
               'forestType',
               projectionLocation.forestType,
               i18n.language,
-            )}
+            )}`}
           </Header>
           <Form>
             <Form.Field>
-              <Form.Radio label="bereits heute mögliche Baumarten" checked />
+              <Form.Radio
+                label="bereits heute mögliche Baumarten"
```

Comment:
Can you please move this string and all other german strings into translation.json?